packages/create-mechanic/function-examples/instagram-story-generator/index.js

```diff
@@ -19,19 +19,18 @@ export const handler = ({ inputs, done, frame, useDrawLoop }) => {
 
   // stuff needed for the looping
   const isPlaying = useRef(true);
-  const frameCount = useDrawLoop(isPlaying.current);
+  const { timestamp } = useDrawLoop(isPlaying.current);
```

Comment:
There's a use of `frameCount`still in the design function so it fails to render :(

Reply:
Oh, good catch. Changed it to use the timestamp here to. I figured that from the initial calculation `frameCount / 15` the rate should be 4 per second (as the function is running at 60fps). So I converted it to `timestamp * 4`.